### DISCUSSION TITLE
bug: slowdown / freeze with large test output

### DISCUSSION THREAD

**Discussion by dvic** [folke/trouble.nvim#293] [Q&A]
I'm using the neotest plugin together with the elixir adapter and I'm facing some performance issues when a failing test outputs a large of amount of text (the ui freezes and becomes unresponsive).

Any idea how I can figure out if the problem lies with `trouble.nvim` or with `neotest`/`neotest-elixir` ?

**Comment by jfrikker-shibumi** [comment]
I'm having a similar issue. I'm using the LazyVim default configuration (more or less). I'm in a fairly large TypeScript project, and I have the references window open. Whenever my cursor stops on the `string` type, the whole editor freezes and uses 100% cpu. I have to kill the whole nvim process to resume working.

Other lsp-based editors report that the string type has 48,000 references.

**Reply by folke** [reply]
That indeed is not great. Looking into this. https://github.com/folke/trouble.nvim/issues/452

**Reply by jfrikker-shibumi** [reply]
The fix worked great! Thanks so much for the fast turnaround!
